dev-erlang/fast_tls/fast_tls-1.0.6.ebuild

```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit rebar
+
+DESCRIPTION="TLS/SSL native driver for Erlang and Elixir"
+HOMEPAGE="https://github.com/processone/fast_tls"
+SRC_URI="https://github.com/processone/${PN}/archive/${PV}.tar.gz
+	-> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~ia64 ~ppc ~sparc ~x86"
+
+DEPEND=">=dev-erlang/p1_utils-1.0.4
+	>=dev-lang/erlang-17.1
+	dev-libs/openssl:0"
```

Comment:
I changed the dependency from `dev-libs/openssl:0` to `dev-libs/openssl:0=`

I also checked libressl support, and that seems to work well, so I've updated the dependency to reflect that too.